dataSets/index.html

```diff
@@ -0,0 +1,68 @@
+<html>
+<head>
+    <!-- Bootstrap 4, Popper, Vue, jQuery, Open Iconic -->
+    <script src="/resources/jquery-3.3.1.js"></script>
+    <link rel="stylesheet" href="/resources/bootstrap-4.0.0-dist/css/bootstrap.min.css"/>
+    <script src="/resources/popper.min.js"></script>
+    <script src="/resources/vue.js"></script>
+    <link href="/resources/open-iconic/font/css/open-iconic-bootstrap.css" rel="stylesheet">
+    <script src="/resources/bootstrap-4.0.0-dist/js/bootstrap.min.js"></script>
+
+    <link href="/global.css" rel="stylesheet">
+</head>
+<body>
+<div class="popupBody">
+    <!-- Title of view -->
+    <header>
+        <a href="/index.html">
+            <span class="oi oi-home"></span>
+        </a>
+        <h3>
+            <span>Data Sets</span>
+        </h3>
+    </header>
+
+    <!-- Dropdown menu -->
```

Comment:
We shouldn't leave commented code in the master branch.

Reply:
you right